text/0000-simple_unwind_annotation.md

```diff
@@ -57,6 +90,41 @@ We should try not to prevent the RFC 2699 or other later RFC from resolving the
 
 - https://github.com/rust-lang/rfcs/pull/2699
 
+The alternatives considered are:
+
+1. Solve the soundness bug by aborting instead of unwinding when reaching an FFI boundary
```

Comment:
This first sentence is not an _alternative_ per se, since this RFC should enable the abort behavior to be stabilized without objections. So perhaps something like:

```suggestion
1. Stabilize the abort-on-FFI-boundary behavior without providing any mechanism to change this behavior.
```